### DISCUSSION TITLE
Network Name on iPhones

### DISCUSSION THREAD

**Discussion by fabioccoelho** [magma/magma#15365] [Q&A]
Hi there. I hope that you're Ok.

I have experience a problem about network name on iPhones. The PLMN ID appears as network name on iPhones Instead the correct network name. Is something that I can do to fix it?

Thank you!

**Comment by edaspb** [comment]
Hi @fabioccoelho, what about SPN field on usim card? Is it working with iPhone?
BTW what plmn do you use for testing?

**Comment by fabioccoelho** [comment]
Hi @edaspb. In SPN field we have the same network name as configured on Magma. We're using MCC and MNC code  724/60. On Android phones, the network name works fine, but on iPhone, just appear the code PLMN ID instead network name.

**Comment by edaspb** [comment]
According to Apple's docs SPN should work. As far as I understand it is the unique method available for Magma's users.
https://support.apple.com/en-mt/guide/deployment/depac6747317/web

BTW may I ask you few not related questions? We are starting to work on Apple devices and got stuck on very initial stage. Our plmn 71070 just doesn't work. How you manage to include yours to Apple carrier config?
And you sad you configured network name on magma, in which part of config it is?

**Comment by fabioccoelho** [comment]
I had write on SIMCARDS the Magma network configurations, like MCC, MNC, etc. About network name on Magma, I just configured on APN config.

**Comment by edaspb** [comment]
What app do you use for usim config?
Regarding APN config, if you talking about APN configuration in NMS GUI that is not a network name. It is an apn name, if I got it right.

**Comment by lucasgonze** [comment]
Closing - this seems to have gone stale.
